apps/server/src/modules/files-storage/entity/filerecord.entity.ts

```diff
@@ -239,4 +239,15 @@ export class FileRecord extends BaseEntityWithTimestamps {
 	public getSchoolId(): EntityId {
 		return this.schoolId;
 	}
+
+	public isPreviewPossible(): boolean {
+		const result =
+			this.mimeType === 'image/bmp' ||
```

Comment:
Können wir aus den ganzen strings ein enum machen? Welches für über openapi auch mit dem nuxt client teilen?

Reply:
if (Object.values(ImageTypes).includes(this.mimeType)) {
    return true
}